src/org/authInfo.ts

```diff
@@ -344,32 +359,50 @@ export class AuthInfo extends AsyncOptionalCreatable<AuthInfo.Options> {
         .map((c) => c.key);
       try {
         const authInfo = await AuthInfo.create({ username });
-        const { orgId, instanceUrl, devHubUsername } = authInfo.getFields();
+        const { orgId, instanceUrl, devHubUsername, expirationDate, isDevHub } = authInfo.getFields();
         final.push({
           aliases,
           configs,
           username,
           instanceUrl,
-          isScratchOrg: Boolean(devHubUsername),
+          isScratchOrg: Boolean(devHubUsername) ?? false,
```

Comment:
I don't think the `?? false` is necessary here since `Boolean` will always return a boolean

```javascript
> Boolean('string')
true
> Boolean('')
false
> Boolean(null)
false
> Boolean(undefined)
false
> Boolean(NaN)
false
> Boolean([])
true
> Boolean({})
true
```